x/smart/README.md

```diff
@@ -0,0 +1,134 @@
+# Smart Dialer
+
+The **Smart Dialer** is a dialer that **automatically chooses the best way to connect** to the internet from a list of configured options.
```

Comment:
```suggestion
The **Smart Dialer** searches for a strategy that unblocks DNS and TLS for a given list of test domains. It takes a config describing multiple strategies, which is used to pick from.
```

Reply:
Maybe something like this, to borrow from the text below (which should probably be updated to remove the duplication).